MultiPoolMiner.ps1

```diff
@@ -83,6 +85,10 @@ $Strikes = 3
 $SyncWindow = 5 #minutes
 
 Set-Location (Split-Path $MyInvocation.MyCommand.Path)
+
+#Make sure config file ends with ".cfg", so no fool will accidentially overwrite MPM files
+$ConfigFile = [io.path]::ChangeExtension($ConfigFile,".cfg")
```

Comment:
This isn't necessary, and will be confusing for people when the file extension randomly gets changed from what they specified.  The code that generates a new config file is in the else of `if (Test-Path $ConfigFile)`, so it can't accidentally overwrite anything.

Reply:
Should it append .txt as default extension (if no extension is given)?